resources/todomvc/architecture-examples/angular/package.json

```diff
@@ -9,8 +9,11 @@
     },
     "scripts": {
         "ng": "ng",
-        "dev": "ng serve",
-        "build": "ng build --base-href /resources/todomvc/architecture-examples/angular/dist/",
+        "dev": "ng serve standalone",
+        "dev:complex": "ng serve complex",
+        "build": "ng build standalone --base-href /resources/todomvc/architecture-examples/angular/projects/standalone/dist/",
+        "build:complex": "gulp build && ng build complex --base-href /resources/todomvc/architecture-examples/angular/projects/complex/dist/",
```

Comment:
This is because when I ran `npm run format`, it formatted `complex/src/index.html`. I've added it to the ignore list and rebuilt it so the output matches what comes out from the build command.

Reply:
no longer relevant. folder structure changed.